fluentd-template.go

```diff
@@ -33,7 +33,7 @@ var fluentDConfigTemplate = `{{- range .IncludeConfigHeadOfFile -}}
     @type  grep
     <regexp>
         key {{ .Key }}
-        pattern {{ .Regex }}
+        pattern /{{.Regex}}/
```

Comment:
hmm i think we should let customer specify the slash.. from https://docs.fluentd.org/filter/grep, "The pattern parameter is string type before 1.2.0." so i don't think we can use slash for version before 1.2

Reply:
Hmm I see, I'll test this filter with >1.2 and <1.2 to verify this is what happens.

Reply:
I tested with and without slashes with v1.0 and v1.5.

v1.5 had the same behavior with and without the slashes, v1.0 only worked without the slashes as expected.